Plugins/Flow.Launcher.Plugin.Calculator/LocalizationHelper/LocalizationConverter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Reflection;
+using System.Windows.Data;
+
+namespace Flow.Launcher.Plugin.Caculator.LocalizationHelper
```

Comment:
how come this and other file is added, dont they already exist?

Reply:
I originally want to remove the dependency on flow.plugin.core here, so move that file here. But it's not able to do that easily so revert the change, but the file is kept unexpectedly.

Reply:
ok lets revert this please if it is unexpected